java/flight/flight-sql/src/test/java/org/apache/arrow/flight/sql/example/FlightSqlExample.java

```diff
@@ -429,6 +453,77 @@ private static VectorSchemaRoot getRoot(final ResultSet data, final BufferAlloca
     return new VectorSchemaRoot(singletonList(dataVector));
   }
 
+  private static VectorSchemaRoot getTypeInfoRoot(CommandGetTypeInfo request, ResultSet typeInfo,
+                                                  final BufferAllocator allocator)
+      throws SQLException {
+    Objects.requireNonNull(allocator, "BufferAllocator cannot be null.");
+
+    VarCharVector typeNameVector = new VarCharVector("type_name", allocator);
```

Comment:
nit, but you might consider `VectorSchemaRoot.create(schema, allocator)` instead of manually building up all the vectors.

Reply:
Add the use of VectorSchemaRoot.create()